website_sale_embargo/README.rst

```diff
@@ -0,0 +1,85 @@
+=====================
+Website Sales Embargo
+=====================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:decb6665d1431cab03aaea40c3e614f451aaad01350e47a9caa2f1d15228fb58
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fodoo--modules-lightgray.png?logo=github
+    :target: https://github.com/OCA/odoo-modules/tree/15.0-add-sales_embargo/website_sale_embargo
+    :alt: OCA/odoo-modules
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/odoo-modules-15-0-add-sales_embargo/odoo-modules-15-0-add-sales_embargo-website_sale_embargo
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/odoo-modules&target_branch=15.0-add-sales_embargo
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module extends the functionality to addd Embargo to Product's HS Code
```

Comment:
Change it to: "This module extends the functionality to add an Embargo between Product's HS Code and country. The confirmation of Sales Orders containing a product with an embargo on the delivery country is prevented.